### ISSUE TITLE
Recommend using .gitattributes, .editorconfig and .gitignore files

### ISSUE THREAD

**Issue by mattpalermo** [spences10/VBA-IDE-Code-Export#75]
These files can be found in the CodeExport project directory. The `.gitattributes` file is important for fixing bugs which arise in Excel due to using Unix style line terminations. The other files help in other less serious ways.

**Comment by spences10** [comment]
This is so that how to use then gets documented?

Bc I have no idea how to use editor config 🙃

**Comment by mattpalermo** [comment]
I don't think the developer even needs to know much about them. At the very least, they just need to know that it will make things work smoother. Editor config is quite simple. Any decent, modern text editor will automatically see the file and adjust its behaviour accordingly. I use the Atom text editor and it works well for me.

**Comment by spences10** [comment]
VS Code here, :heart: it  :+1:

**Comment by mattpalermo** [comment]
Nice! I remember now, I saw some VBA language plugin for vs code on your
github. I've heard good things about it. I'll try it out one day.

**Comment by spences10** [comment]
Yeah I had a play around with making some snippets and then syntax highlighting and then some icons for the editor:

**syntax highlighting**
[https://github.com/spences10/vscode-vba](https://github.com/spences10/vscode-vba)

**snippets** 
[https://github.com/spences10/vba-snippets](https://github.com/spences10/vba-snippets)

**icons** 
[https://github.com/spences10/vscode-vba-icons](https://github.com/spences10/vscode-vba-icons)

If you're interested :smile: :ok_hand: 

**Anyhoo**

Back to #75 does this need to be mentioned in the documentation somewhere?

**Comment by mattpalermo** [comment]
Yea, I think it does need to be documented. Probably the "Using" section of the README is the best place?
